src/perl/Bio/EnsEMBL/Pipeline/PipeConfig/BRC4_genome_loader_conf.pm

```diff
@@ -350,12 +339,11 @@ sub pipeline_analyses {
       -module      => 'Bio::EnsEMBL::Hive::RunnableDB::SystemCmd',
       -parameters  => {
         cmd => 'mkdir -p #log_path#; '
-             . 'echo "checking #json# against #schema#" > #log_path#/#metadata_type#.log; '
-             . 'schemas_json_validate --json_file #json# --json_schema #schema# '
+             . 'echo "checking #json# against #metadata_type#" > #log_path#/#metadata_type#.log; '
+             . 'schemas_json_validate --json_file #json# --json_schema #metadata_type# '
```

Comment:
just a note.`metadata_type` -> `JSON file` mapping is done in `schemas_json_validate` itself https://github.com/Ensembl/ensembl-genomio/blob/fb33daabb7502fb25b240654988c506c7ddec2de/src/python/ensembl/io/genomio/schemas/json/validate.py#L45

Reply:
Do you want me to add this comment in the code? Or just leaving this here as a future reference.

Reply:
m.b. we need this in the documentation